tests/test_contrib_orcid.py

```diff
@@ -128,6 +132,7 @@ def test_authorized_signup(userprofiles_app, example_orcid, orcid_bio):
             id=example_data['orcid']
         ).one()
         # FIXME see contrib/orcid.py line 167
+        assert user.profile.full_name == 'Josiah Carberry'
```

Comment:
@jirikuncar  If I understood correctly that's the place for the check. The problem is the `full_name` is empty. I am a bit lost in the packages, but if I get it right `invenio-userprofiles` should use the `account_info` function to populate the profile (resulting to 'Josiah Carberry' filling the `full_name` field).